cw-orch-daemon/src/core.rs

```diff
@@ -349,32 +347,18 @@ impl<Sender: TxSender> DaemonAsyncBase<Sender> {
 
         log::debug!(target: &transaction_target(), "Uploading file at {:?}", wasm_path);
 
-        let file_contents = std::fs::read(wasm_path.path())?;
-        let mut e = write::GzEncoder::new(Vec::new(), Compression::default());
```

Comment:
So think this code should be left untouched as well.

Reply:
Ok, reverting [6b0742c](https://github.com/AbstractSDK/cw-orchestrator/pull/487/commits/6b0742ce0aa5860dda9cb8a40a7e47f466cdceb6) and creating a function to avoid code duplication